fabtests/man/fabtests.7.md

```diff
@@ -217,9 +220,9 @@ testing scope is limited.
 This test runs a series of tests over multiple formats and patterns to help
 validate at scale. The patterns are an all to all, one to all, all to one and
 a ring. The tests also run accross multiple capabilites, such as messages, rma,
-atomics, and tagged messages. Currently, there is no option to run these 
```

Comment:
Minor nit, can the whitespace changes be put into a different commit?

Reply:
Yep. Will do.